### PR TITLE
Add `rabbit` & `turtle`/`snail` & `shell`/aminal icons

### PR DESCRIPTION
**Pull request by danielbayley**
Can represent speed, fast/slow (tortoise/hare…)

Untitled

Related to `squirrel` in #1229.

Alts:
Artboard 1@2x

Untitled

<br/>
<br/>

Artboard 1@2x
<br/>
<br/>
Untitled
<br/>
<br/>

Updated to better match the Lucide logo, and `lollipop` in #1149.

### CODE REVIEW

**Comment by jguddas** [comment]
We definitely need both `rabbit` and `rabit-2`, extremely cute, `turtle` not so much.

**Comment by karsa-mistmere** [comment]
The rabbit is suuuuuper cute. 🐰
I'd vote for the one with the lopsided ears, it's much more rabbity.

Also not sure about the turtle I'm afraid.

File: icons/rabbit.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M20 8.5V4a2 2 0 0 0-4 0v3" />
+  <path d="M16 21v-2.3a2.7 2.7 0 0 1 2.7-2.7H20a2 2 0 0 0 2-2v-1c0-3.3-2.7-6-6-6l-3.6-3.4a1.93 1.93 0 1 0-2.8 2.8L13 10a7 7 0 0 0-7 7 4 4 0 0 0 4 4h8" />
+  <path d="M7.6 12.5A3 3 0 1 0 5 17c.4 0 .7-.1 1-.2" />
+  <path d="M15.7 21a3.5 3.5 0 0 0-3.2-5" />
+  <path d="M18 12h.01" />
```
Comment (jguddas) [inline icons/rabbit.svg:16]:
```suggestion
  <path d="M20 8.5V4a2 2 0 0 0-4 0v3" />
  <path d="M16 21v-2.3a2.7 2.7 0 0 1 2.7-2.7H20a2 2 0 0 0 2-2v-1c0-3.3-2.7-6-6-6l-3.6-3.4a1.93 1.93 0 1 0-2.8 2.8L13 10a7 7 0 0 0-7 7 4 4 0 0 0 4 4h8" />
  <path d="M 7.6 12.5 A3 3 0 1 0 6 16.8" />
  <path d="M15.7 21a3.5 3.5 0 0 0-3.2-5" />
  <path d="M18 12h.01" />
```

**Comment by ericfennis** [comment]
Really nice, love the rabbit.
I agree with @jguddas @karsa-mistmere, but to be fair it works on 100%.

**Comment by danielbayley** [comment]
> to be fair it works on 100%.

I like the turtle to be honest, but just added `snail`, as another alternative for ‘slow’. Although it does kind of ruin the parable of the tortoise/hare thing…

@ericfennis @jguddas @karsa-mistmere See what you guys think…

**Comment by karsa-mistmere** [comment]
Oh no. 🙈

Snail shells never curl in that way:
image

**Comment by danielbayley** [comment]
> Snail shells never curl that way

@karsa-mistmere Fixed.

**Comment by karsa-mistmere** [comment]
The snail looks a bit too large and I think it looks a bit weird that the shell doesn't fully connect to its body. Almost like a slug masquerading as a snail. 😄

**Comment by jguddas** [comment]
I like the snail.

**Comment by danielbayley** [comment]
> I like the snail.

Yeah think I **s***nailed* it (sorry 🙈) with these…

I like the existing one, but here are some alts anyway—`snail` and `shell`:

Artboard 1@2x
<br/>
<br/>
Untitled

**Comment by karsa-mistmere** [comment]
I really like row no. 3 in the last two columns, I think they give the best vibes for the "slow" use case, the previous columns are maybe too energetic in comparison.

**Comment by ericfennis** [comment]
I'm ok with these changes. Should the snail not be flipped? So the direction is the same as the other icons?

**Comment by jguddas** [comment]
I still feel like the turtle needs some work, but the other ones are super cute :heart:

**Comment by danielbayley** [comment]
> Should the snail not be flipped? So the direction is the same as the other icons?

@ericfennis So the reason `snail` (and `shell`) are that way is to match `lollipop` in https://github.com/lucide-icons/lucide/pull/1149#issuecomment-1520066481 and the Lucide logo itself (even though I know it’s not an actual icon…) Again, going for consistency.

Untitled
<br/>
<br/>

> I still feel like the turtle needs some work

@jguddas How about something like this…?

preview

**Comment by jguddas** [comment]
My failed attempt at a cute turtle icon (88).

Maybe something like this?
image

**Comment by danielbayley** [comment]
> Maybe something like this?

@jguddas I don't think it represents slow, as it was meant to, if it's not on land…

**Comment by jguddas** [comment]
> > Maybe something like this?
> 
> @jguddas I don't think it represents slow, as it was meant to, if it's not on land…

We do have the snail 🐌.

**Comment by jguddas** [comment]
> I'm ok with these changes. Should the snail not be flipped? So the direction is the same as the other icons?

I think as a guideline we should make icons that flow with text, so ltr.

Maybe this is just an Arabic snail tho.

**Comment by karsa-mistmere** [comment]
Definitely would prefer a ltr snail, especially if it's meant to be used next to rabbit.

File: icons/paw-print.svg
Comment (karsa-mistmere) [inline icons/paw-print.svg:1]: Nice! I also have a paw print in my drafts, but yours is much cleaner.
image

**Review by ericfennis — APPROVED** [review]

File: icons/turtle.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M4.7 8a8.26 8.26 0 0 0 10.6 0" />
+  <path d="M14 14a4 4 0 0 0-8 0v3c0 .6-.4 1-1 1H3c-.6 0-1-.4-1-1v-3a8 8 0 0 1 16 0v3c0 .6-.4 1-1 1h-2c-.6 0-1-.4-1-1Z" />
+  <path d="M17 10h3a2 2 0 0 1 0 4H2" />
```
Comment (karsa-mistmere) [inline icons/turtle.svg:14]: The shell feels a bit off when scaled up, how about?
image

```suggestion
  <path d="M8 10h4" />
  <path d="m15 8-3 2 2 4v3c0 .6.4 1 1 1h2c.6 0 1-.4 1-1v-3a8 8 0 0 0-16 0v3c0 .6.4 1 1 1h2c.6 0 1-.4 1-1v-3l2-4-3-2" />
  <path d="M17 10h3a2 2 0 0 1 0 4H2" />
```

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

File: icons/paw-print.svg [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="11" cy="4" r="2" />
+  <circle cx="18" cy="8" r="2" />
+  <circle cx="20" cy="16" r="2" />
+  <path d="M14 14a4 4 0 0 0-4-4H5.5a3.5 3.5 0 0 0-1.25 6.75c2 .75 2.25 1 3 3A3.5 3.5 0 0 0 14 18.5Z" />
```
[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/paw-print.svg]: > tedious manual optimization

It does get that way 😅

File: icons/turtle.svg [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m14 14-2-4H8l-2 4v3c0 .6-.4 1-1 1H3c-.6 0-1-.4-1-1v-3a8 8 0 0 1 16 0v3c0 .6-.4 1-1 1h-2c-.6 0-1-.4-1-1Z" />
+  <path d="M4.8 7.9 8 10" />
+  <path d="m12 10 3.2-2.1" />
+  <path d="M16.9 10H20a2 2 0 0 1 0 4H2" />
```
Comment (karsa-mistmere) [inline icons/turtle.svg:15]: Overoptimized again:
```suggestion
<path d="m12 10 2 4v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3a8 8 0 0 0-16 0v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3l2-4h4Z" />
  <path d="M4.8 7.9 8 10" />
  <path d="m12 10 3.2-2.1" />
  <path d="M16.93 10H20a2 2 0 0 1 0 4H2" />
```

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

File: icons/turtle.svg (hunk 4 above)
Comment (danielbayley) [reply icons/turtle.svg]: Ah was just fixing the lines on it’s back… https://lucide-site-git-feat-added-edit-page-jguddas.vercel.app/edit can be a bit too keen! :)

File: icons/turtle.svg [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m12 10 2 4v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3a8 8 0 0 0-16 0v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3l2-4h4Z" />
+  <path d="M4.824 7.9 8 10" />
+  <path d="M15.176 7.9 12 10" />
+  <path d="M16.9 10H20a2 2 0 0 1 0 4H2" />
```
Comment (karsa-mistmere) [inline icons/turtle.svg:15]: Still not quite right (although mea culpa, the previous one I submitted was also wrong 🤦‍♂️):
```suggestion
  <path d="m12 10 2 4v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3a8 8 0 1 0-16 0v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3l2-4h4Z" />
  <path d="M8 10 4.81 7.9" />
  <path d="m12 10 3.19-2.1" />
  <path d="M16.93 10H20a2 2 0 0 1 0 4H2" />
```

**Review by karsa-mistmere — APPROVED** [review]

**Comment by garikAsplund** [comment]
I want to use the paw print one, but only having 3 pads instead of 4 is so weird!! Can you make it like an actual dog paw please??

**Pull request merged**